quodlibet/quodlibet/qltk/controls.py

```diff
@@ -131,15 +135,20 @@ def __init__(self, player):
             item = RadioMenuItem(group=item, label=title,
                                  use_underline=True)
             rg.append(item)
-            item.connect("toggled", self.__changed, player, profile)
-            if player.replaygain_profiles[0] == profile:
+            item.connect("toggled", self.__changed, player, mode)
+            if replaygain_mode == mode:
                 item.set_active(True)
             item.show()
 
-    def __changed(self, item, player, profile):
+    def __set_mode(self, player, mode):
+        player.replaygain_profiles[0] = \
```

Comment:
Please handle unknown mode values. Crashing in this case makes it hard to extend them and switching between QL versions.

(In general, code should handle garbage config values in some "sane" way if possible)